splitio/engine/grammar/constants.go

```diff
@@ -7,10 +7,10 @@ const (
 	SplitStatusArchived = "ARCHIVED"
 
 	// SplitAlgoLegacy represents the legacy implementation of hash function for bucketing
-	SplitAlgoLegacy = 0
+	SplitAlgoLegacy = 1
```

Comment:
oh boy

Reply:
is it possible to add a unit test here to make sure the runtime is picking up the right algo?